src/calculator.js

```diff
@@ -0,0 +1,31 @@
+class Calculator {
+  constructor(num1, num2) {
+    this.num1 = num1;
+    this.num2 = num2;
+  }
+
+  add() {
+    return this.num1 + this.num2;
+  }
+
+  subtract() {
+    return this.num1 - this.num2;
+  }
+
+  multiply() {
+    return this.num1 * this.num2;
+  }
+
+  divide() {
+    return this.num1 / this.num2;
+  }
+}
```

Comment:
ideally calculator should have methods that control their own behaviour, in the sense that, add, subtract,.... should be able to take numbers themselves and manipulate them, currently you are passing the numbers to calculator and calling the operation on it

Reply:
Ideally, a problem can be solved in many ways there is no one particular way of solving algorithm. your suggestion is good, but, I prefer class over functions any day.